EpiResponsivePicture/TagBuilders/PictureTagBuilder.cs

```diff
@@ -0,0 +1,155 @@
+        private ResizedPictureViewModel pictureViewModel;
+        private FocalPoint focalPoint;
+        private string pictureUrl;
+        private string altImgText;
+        
+        private PictureTagBuilder()
+        {
+            element = new TagBuilder("picture");
+        }
+
+        public static PictureTagBuilder Create() => new();
+
+        public PictureTagBuilder WithProfile(PictureProfile profile)
+        {
+            pictureProfile = profile;
+            return this;
+        }
+
+        public PictureTagBuilder WithContentReference(ContentReference contentReference)
+        {
+            pictureContentReference = contentReference;
+            return this;
+        }
+
+        public PictureTagBuilder WithFallbackUrl(string fallbackUrl)
+        {
+            pictureFallbackUrl = fallbackUrl;
+            return this;
+        }
+
+        public PictureTagBuilder WithViewModel(ResizedPictureViewModel viewModel)
+        {
+            pictureViewModel = viewModel;
+            return this;
+        }
+
+        public TagBuilder Build()
+        {
+            Guard.IsNotNull(pictureProfile, nameof(pictureProfile));
+            Guard.IsNotNull(pictureContentReference, nameof(pictureContentReference));
+            
+            pictureFallbackUrl ??= string.Empty;
+            pictureViewModel ??= new ResizedPictureViewModel();
+            
+            GetPictureData();
+
+            var sourceTagBuilder = SourceTagBuilder
+                .Create()
+                .WithImageUrl(pictureUrl)
+                .WithFocalPoint(focalPoint)
+                .WithResizedImageFormat(pictureProfile.Format);
+
+            foreach (var pictureSource in pictureProfile.Sources)
+            {
+                var sourceTag = sourceTagBuilder.WithSource(pictureSource).Build();
+
+                element.InnerHtml.AppendHtml(sourceTag.RenderSelfClosingTag());
+                
+                sourceTagBuilder.Clear();
+            }
+            
+            AddAdditionalAttributes();
+
+            var imgTag = CreateImageElement();
+            element.InnerHtml.AppendHtml(imgTag.RenderSelfClosingTag());
+            
+            return element;
+        }
+        
+        private void GetPictureData()
+        {
+            var imageFound = ServiceLocator.Current.GetInstance<IContentLoader>().TryGet<IContentData>(pictureContentReference,
+                new LoaderOptions { LanguageLoaderOption.FallbackWithMaster() }, out var content) && content is IImage;
+            
+            if(imageFound)
+                GetImgAltText((IImage) content);
+
+            GetFocalPoint(content as IResponsiveImage);
+            SetPictureProfilesAspectRatios(content as IResponsiveImage);
+
+            pictureUrl = imageFound ? ResolveUrl() : pictureFallbackUrl;
+        }
+
+        private void GetImgAltText(IImage image)
+        {
+            altImgText = image.Description ?? string.Empty;
```

Comment:
Change variable name to `altImageText` for consistency

Shall alt fallback to some meaningful value like 'image'? On the one hand it is recommended to have alt tag on the other hardcoded value will not be locale specific. Maybe mark `Description` property in `IImage` as `[Required]`?